features/advanced-security-settings.md

```diff
@@ -96,4 +96,20 @@ Using the blocklist will block domains that are hotlinking your content or asset
 
 The allowlist can be used to allow only selected domains to access your assets. For example, you can use it to serve content only on your allowed domains and automatically block other referrers.
 
-If you do not want any HTTP referrer restriction, you can opt for no restriction, which is the default behavior.
\ No newline at end of file
+If you do not want any HTTP referrer restriction, you can opt for no restriction, which is the default behavior.
+
+## User Agent based restriction
```

Comment:
Add this feature as the 5th point in the list at the top of this page.

Reply:
Added in 6cab65d1b333c9bb94cccaadf6c06d5f63a57f27